test/acceptance/ast/html/block/annotated_block_test.exs

```diff
@@ -75,9 +75,9 @@ defmodule Acceptance.Ast.Html.Block.AnnotatedBlockTest do
     test "even non-closed block elements" do
       markdown = "<div>\n```elixir\ndefmodule Mine do\n</div>"
       ast = [vtag("div", ["```elixir", "defmodule Mine do"])]
-      messages = []
+      messages = [{:warning, 1, "Failed to find closing <div>"}]
 
-      assert as_ast(markdown, annotations: @annotations) == {:ok, ast, messages}
+      assert as_ast(markdown, annotations: @annotations) == {:error, ast, messages}
```

Comment:
It is the same AST but note the AST already differs from GitHub. So I am fine with emitting a warning to try to force users to fix up their markdown.

Reply:
ok with me